evmrpc/info.go

```diff
@@ -124,7 +124,7 @@ func (i *InfoAPI) FeeHistory(ctx context.Context, blockCount math.HexOrDecimal64
 
 type GasAndReward struct {
 	GasUsed uint64
-	Reward  uint64
```

Comment:
why was this converted to a big int?

Reply:
because GasPrice will also be in `sei-wei` which might cause overflow with uint64